cmake.deps/cmake/TreesitterCMakeLists.txt

```diff
@@ -5,7 +5,7 @@ add_compile_options(-w)
 
 add_library(tree-sitter lib/src/lib.c)
 target_include_directories(tree-sitter
-  PRIVATE lib/src lib/include)
+  PRIVATE lib/src lib/src/wasm lib/include)
```

Comment:
For the record, this is only needed to pick up https://github.com/tree-sitter/tree-sitter/blob/master/lib/src/wasm/stdlib-symbols.txt (list of exported symbols that are safe in wasm) during build.

Tree-sitter's own `build.rs` simply [copies that file](https://github.com/tree-sitter/tree-sitter/blob/3095fbe07bbcc605910b7ad8bbd6313e0a07e678/lib/binding_rust/build.rs#L9-L13). Would this be a (better) alternative? @dundargoc

Reply:
That's basically what this does, "including" a directory here just means the directory will be copied during installation.

Reply:
My question was whether it would make sense to only copy the _file_, not the whole directory?

Reply:
I don't think that's possible. I can't come up with a way at the top of my head.